src/app/meteo/form-filter/form-filter.component.html

```diff
@@ -0,0 +1,114 @@
+<section class="container">
+    <header class="page-header">
+        <h2>Filter table</h2>
+    </header>
+
+    <div class="row">
+        <form class="form-filter col-md-6 col-sm-9"
+              [formGroup]="filterForm"
+              (submit)="applyFilters(filterForm)">
+
+            <div class="form-group" formGroupName="columns">
+                <label>Show columns:</label> <br/>
+                <label class="checkbox-inline" *ngFor="let filter of filterList">
+                    <input type="checkbox"
+                           value="filter"
+                           [formControlName]="filter">
+                    {{filter}}
+                </label>
+            </div>
+
+            <div class="form-group row"
+                 [ngClass]="{'has-error': filterForm.get('temperature').dirty&& filterForm.get('temperature').errors,
```

Comment:
@LeraKruchek I know, and will change it. However, it's a bit different. I mean ng-dirty aplies to input element, has-error, has-success classes to form-group element (parent container).